xtend-website/_posts/releasenotes/2019-06-04-version-2-18-0.md

```diff
@@ -46,6 +46,38 @@ def m() {
 }
 ```
 
+
+## Eclipse Integration
+
+### New Quickfix: Rename Class
+
+When an Xtend class name does not match the file name it is defined in then Xtend raises a warning. A new quickfix has been provided to rename the class according to the file name. ([#744](https://github.com/eclipse/xtext-xtend/issues/744))
+
+![Quickfix: Rename Class]({{site.baseurl}}/images/releasenotes/2_18_Quickfix_RenameClass.gif){:height="50%" width="50%"}
+
+### Quickfix "Remove Unused Member" Improved
+
+The quickfix "Remove member" for unused members was improved. Additionally to the member itselfs the quickfix will now also remove obsolete import statements write accesses. ([xtext-xtend](https://github.com/eclipse/xtext-xtend/issues/736))
```

Comment:
and is missing: remove the obsolete import statements **and** the write accesses

Reply:
I don't think that adding *the* is correct here. Added *and* for next commit.